blazor/speeddial/radial-menu.md

```diff
@@ -46,7 +46,7 @@ The SpeedDial angle starts with `0` degree in the right side and increases in th
 @using Syncfusion.Blazor.Buttons
 
 <SfSpeedDial Position="FabPosition.MiddleCenter" Mode="SpeedDialMode.Radial" OpenIconCss="e-icons e-edit">
```

Comment:
Use the Middle Right position and angle as 120 - 240. Update the image as well.

Reply:
Changes Updated.